pkgs/top-level/python-packages.nix

```diff
@@ -5992,6 +5992,8 @@ in {
 
   pygame_sdl2 = callPackage ../development/python-modules/pygame_sdl2 { };
 
+  pygame-gui = callPackage ../development/python-modules/pygame-gui { };
```

Comment:
Is this a library or an end user program?

Reply:
This is a library